client/src/components/TreeImageScrubber.js

```diff
@@ -87,6 +88,9 @@ function TreeImageScrubber({ classes, getScrollContainerRef, ...props }) {
     approveTreeImage(id)
       .then(result => {
         dispatch({ type: "approveTreeImage", id });
+        if (needtoLoadMoreTreeImages()) {
+          // loadMoreTreeImages();
```

Comment:
this is a remnant of how i first tried to do the check for having no images. 
I will delete, since the check is now done whenever state changes, not just in approve/reject.